tests/src/test/java/com/google/cloud/BomContentTest.java

```diff
@@ -116,7 +116,11 @@ private static String buildMavenCentralUrl(Artifact artifact) {
         + artifact.getArtifactId()
         + "/"
         + artifact.getVersion()
-        + "/";
+        + "/"
+        + artifact.getArtifactId()
+        + "-"
+        + artifact.getVersion()
+        + ".pom";
```

Comment:
I feel this change is no op but no harm getting the pom file. The check is passing.